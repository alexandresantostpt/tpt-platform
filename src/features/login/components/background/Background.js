import React from 'react'
import styled from 'styled-components'

import bgLogin from '@img/background/login.png'

const StyledBg = styled.div`
    background: url(${bgLogin}) no-repeat bottom / cover;
    bottom: 0;
    height: 100vh;
    opacity: 0.1;
    position: absolute;
    width: 100vw;
`

const Background = () => <StyledBg />

export default Background
