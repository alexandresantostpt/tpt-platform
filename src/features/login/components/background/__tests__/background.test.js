import React from 'react'
import renderer from 'react-test-renderer'

import Background from '../Background'

describe('Tests for Background component', () => {
    it('Should rendering without crash', () => {
        const tree = renderer.create(<Background />).toJSON()
        expect(tree).toMatchSnapshot()
    })
})
