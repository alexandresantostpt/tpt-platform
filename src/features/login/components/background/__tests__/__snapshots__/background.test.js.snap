// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Tests for Background component Should rendering without crash 1`] = `
<div
  className="Background__StyledBg-sc-904t04-0 bTqgFQ"
/>
`;
